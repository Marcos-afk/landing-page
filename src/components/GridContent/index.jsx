import * as Styled from './styles';
import P from 'prop-types';

const GridContent = ({ children }) => {
  return <Styled.Container>{children}</Styled.Container>;
};

GridContent.propTypes = {
  children: P.node.isRequired,
};

export default GridContent;
