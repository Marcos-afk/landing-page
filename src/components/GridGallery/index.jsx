import * as Styled from './styles';
import P from 'prop-types';
import SectionBackground from '../SectionBackground';
import Heading from '../Heading';
import TextComponent from '../TextComponent';

const GridGallery = ({ background = false, title, description, grid }) => {
  return (
    <SectionBackground background={background}>
      <Styled.Container>
        <Heading size="huge" uppercase colorDark={!background} as="h2">
          {title}
        </Heading>
        <TextComponent>{description}</TextComponent>
        <Styled.Grid>
          {grid.map(element => (
            <Styled.GridElement key={`${element.altText}${element.srcImg}`}>
              <Styled.Image src={element.srcImg} alt={element.altText} />
            </Styled.GridElement>
          ))}
        </Styled.Grid>
      </Styled.Container>
    </SectionBackground>
  );
};

GridGallery.propTypes = {
  title: P.string.isRequired,
  description: P.string.isRequired,
  grid: P.arrayOf(
    P.shape({
      altText: P.string.isRequired,
      srcImg: P.string.isRequired,
    }),
  ).isRequired,
  background: P.bool,
};

export default GridGallery;
