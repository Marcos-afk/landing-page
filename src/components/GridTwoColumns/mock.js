const data = {
  title: 'teste de grid de duas colunas',
  text: 'grid two columns',
  srcImg: 'https://res.cloudinary.com/dlizakp2a/image/upload/v1613749289/javascript_b57bf48cda.svg',
};

export default data;
