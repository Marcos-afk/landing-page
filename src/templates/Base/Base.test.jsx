import Template from './';
import footerMock from '../../components/Footer/mock';
import menuMock from '../../components/NavLinks/mock';
import logoMock from '../../components/LogoLink/mock';
import GridMock from '../../components/GridText/mock';
import GridText from '../../components/GridText';
import GridTwoColumns from '../../components/GridTwoColumns/';
import GridTwoColumnsMock from '../../components/GridTwoColumns/mock';
import GridGallery from '../../components/GridGallery';
import GridGalleryMock from '../../components/GridGallery/mock';
import GridContent from '../../components/GridContent';
import GridContentMock from '../../components/GridContent/mock';
import { renderTheme } from '../../styles/render-theme';

describe('< Template />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <Template footerHtml={footerMock.footerHtml} links={menuMock} logoData={logoMock}>
        <GridTwoColumns {...GridTwoColumnsMock} background />
        <GridContent {...GridContentMock} />
        <GridText {...GridMock} background />
        <GridGallery {...GridGalleryMock} />
        <GridText {...GridMock} background />
        <GridContent {...GridContentMock} />
      </Template>,
    );
    expect(container).toMatchSnapshot();
  });
});
