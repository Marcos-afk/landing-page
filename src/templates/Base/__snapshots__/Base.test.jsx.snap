// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< Template /> should render 1`] = `
.c17 {
  font-size: 2.4rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c30 {
  text-align: center;
  border-top: 0.1rem solid;
}

.c30 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c30 .c16 {
  font-size: 1.6rem;
}

.c30 .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c8 > img {
  max-height: 3rem;
}

.c7 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c15 {
  color: #FFFFFF;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c20 {
  color: #0A1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c25 {
  color: #FFFFFF;
  font-size: 3.2rem;
  text-transform: none;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  border-bottom: #DDDDDD;
  background: #FFFFFF;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 .c6 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  width: 4rem;
  right: 4rem;
  background: #0A1128;
  color: #FFFFFF;
  border: none;
  display: none;
  pointer-events: all;
}

.c0 > svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c10 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0.75rem;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #DC143C;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c31 {
  position: fixed;
  background: #0A1128;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c11 {
  padding-top: 4.5rem;
}

.c22 .c16 {
  margin-bottom: 6.4rem;
}

.c23 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  overflow: hidden;
  width: 100%;
}

.c24 .c6 {
  position: relative;
  left: 5rem;
}

.c24 .c6::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 7rem;
  top: -3rem;
  left: -5rem;
  -webkit-transform: rotate(5deg);
  -ms-transform: rotate(5deg);
  transform: rotate(5deg);
}

.c12 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c18 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c13 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 3.2rem;
}

.c13 .c6 {
  margin-bottom: 4.0rem;
}

.c26 > .c16 {
  margin-bottom: 6.4rem;
}

.c27 {
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c28 {
  overflow: hidden;
}

.c29 {
  width: 100%;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c29:hover {
  -webkit-transform: scale(1.2) rotate(10deg);
  -ms-transform: scale(1.2) rotate(10deg);
  transform: scale(1.2) rotate(10deg);
}

.c19 {
  text-align: center;
  max-width: 58rem;
  margin: 0 auto;
}

.c21 {
  margin: 6.4rem 0;
}

@media (max-width:768px) {
  .c15 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c20 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: auto;
  }

  .c2 .c6 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c9 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c11 {
    padding-top: 0;
  }
}

@media (max-width:768px) {
  .c23 {
    grid-template-columns: 1fr;
  }
}

@media (max-width:768px) {
  .c13 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width:768px) {
  .c14 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <button
    aria-label="Open/Close menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="http://localhost:3000/"
          >
            <img
              alt="LOGO"
              src="https://files.passeidireto.com/1cc481c0-d28a-4f01-814f-1d9c0d3d2a26/1cc481c0-d28a-4f01-814f-1d9c0d3d2a26.jpeg"
            />
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=Bo8Vu8YI8WY"
            target="_blank"
          >
            Industry Baby
          </a>
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=tqHMj-m_mDs"
            target="_self"
          >
            Wwe
          </a>
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=bzda5UA71Y8"
            target="_self"
          >
            Devil May Cry 2
          </a>
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=bzda5UA71Y8"
            target="_self"
          >
            Devil May Cry 3
          </a>
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=bzda5UA71Y8"
            target="_self"
          >
            Devil May Cry 4
          </a>
          <a
            class="c10"
            href="https://www.youtube.com/watch?v=bzda5UA71Y8"
            target="_self"
          >
            Devil May Cry 5
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <div
            class="c14"
          >
            <h2
              class="c6 c15"
            >
              teste de grid de duas colunas
            </h2>
            <div
              class="c16 c17"
            >
              grid two columns
            </div>
          </div>
          <div
            class=""
          >
            <img
              alt="teste de grid de duas colunas"
              class=""
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1613749289/javascript_b57bf48cda.svg"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="c18"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c19"
        >
          <h2
            class="c6 c20"
          >
            teste de grid de duas colunas
          </h2>
          <div
            class="c21"
          >
            <div
              class="c16 c17"
            >
              <p>
                <strong>
                  Pellentesque habitant morbi tristique
                </strong>
                 senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. 
                <em>
                  Aenean ultricies mi vitae est.
                </em>
                 Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, 
                <code>
                  commodo vitae
                </code>
                , ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. 
                <a
                  href="#"
                >
                  Donec non enim
                </a>
                 in turpis pulvinar facilisis. Ut felis.
              </p>
              

  
              <h2>
                Header Level 2
              </h2>
              

  
              <ol>
                
     
                <li>
                  Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
                </li>
                
     
                <li>
                  Aliquam tincidunt mauris eu risus.
                </li>
                
  
              </ol>
              

  
              <blockquote>
                <p>
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus magna. Cras in mi at felis aliquet congue. Ut a est eget ligula molestie gravida. Curabitur massa. Donec eleifend, libero at sagittis mollis, tellus est malesuada tellus, at luctus turpis elit sit amet quam. Vivamus pretium ornare est.
                </p>
              </blockquote>
              

  
              <h3>
                Header Level 3
              </h3>
              

  
              <ul>
                
     
                <li>
                  Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
                </li>
                
     
                <li>
                  Aliquam tincidunt mauris eu risus.
                </li>
                
  
              </ul>
              

  
              <pre>
                <code>
                  
  #header h1 a {
    display: block;
    width: 300px;
    height: 80px;
  }
  
                </code>
              </pre>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c22"
        >
          <h2
            class="c6 c15"
          >
            My grid
          </h2>
          <div
            class="c16 c17"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c23"
          >
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c18"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c26"
        >
          <h2
            class="c6 c20"
          >
            My grid
          </h2>
          <div
            class="c16 c17"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c27"
          >
            <div
              class="c28"
            >
              <img
                alt="Teste 1"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=1"
              />
            </div>
            <div
              class="c28"
            >
              <img
                alt="Teste 2"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=2"
              />
            </div>
            <div
              class="c28"
            >
              <img
                alt="Teste 3"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=3"
              />
            </div>
            <div
              class="c28"
            >
              <img
                alt="Teste 4"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=4"
              />
            </div>
            <div
              class="c28"
            >
              <img
                alt="Teste 5"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=5"
              />
            </div>
            <div
              class="c28"
            >
              <img
                alt="Teste 6"
                class="c29"
                src="https://source.unsplash.com/random/800x800?r=6"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c22"
        >
          <h2
            class="c6 c15"
          >
            My grid
          </h2>
          <div
            class="c16 c17"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c23"
          >
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c24"
            >
              <h3
                class="c6 c25"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c18"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c19"
        >
          <h2
            class="c6 c20"
          >
            teste de grid de duas colunas
          </h2>
          <div
            class="c21"
          >
            <div
              class="c16 c17"
            >
              <p>
                <strong>
                  Pellentesque habitant morbi tristique
                </strong>
                 senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. 
                <em>
                  Aenean ultricies mi vitae est.
                </em>
                 Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, 
                <code>
                  commodo vitae
                </code>
                , ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. 
                <a
                  href="#"
                >
                  Donec non enim
                </a>
                 in turpis pulvinar facilisis. Ut felis.
              </p>
              

  
              <h2>
                Header Level 2
              </h2>
              

  
              <ol>
                
     
                <li>
                  Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
                </li>
                
     
                <li>
                  Aliquam tincidunt mauris eu risus.
                </li>
                
  
              </ol>
              

  
              <blockquote>
                <p>
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus magna. Cras in mi at felis aliquet congue. Ut a est eget ligula molestie gravida. Curabitur massa. Donec eleifend, libero at sagittis mollis, tellus est malesuada tellus, at luctus turpis elit sit amet quam. Vivamus pretium ornare est.
                </p>
              </blockquote>
              

  
              <h3>
                Header Level 3
              </h3>
              

  
              <ul>
                
     
                <li>
                  Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
                </li>
                
     
                <li>
                  Aliquam tincidunt mauris eu risus.
                </li>
                
  
              </ul>
              

  
              <pre>
                <code>
                  
  #header h1 a {
    display: block;
    width: 300px;
    height: 80px;
  }
  
                </code>
              </pre>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c30"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c16 c17"
        >
          <p>
            <a
              href="https://www.linkedin.com/in/marcos-melo-94609b1b6/"
            >
              Feito com ❤ por Marcos André
            </a>
          </p>
        </div>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c31"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
